javascript/javascript2/week2/movies.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <h1>Hello World!</h1>
+    <script src="movies.js"></script>
+  </body>
```

Comment:
These file is not relevant for this homework. Please remove it it is confusing and useless for this task.

Reply:
I was using this file for 'Working with movies' part of the homework. I guess, js file of it is not loading because of having a large array in the file.